app/models/expense.rb

```diff
@@ -146,6 +146,15 @@ def distance_gt_calculated?
     distance > calculated_distance
   end
 
+  def establishment
+    return unless expense_type.car_travel?
+    Establishment.find_by(name: location)
+  end
+
+  def establishment_category
```

Comment:
just thinking. do we need this as the establishment method mimic a has_one relationship so:

```
expense.establishment&.category
```